parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -448,4 +449,16 @@ private void appendHex(byte[] array, int offset, int length, StringBuilder build
       }
     }
   };
+
+  static final PrimitiveStringifier FLOAT16_STRINGIFIER = new BinaryStringifierBase("FLOAT16_STRINGIFIER") {
+
+    @Override
+    String stringifyNotNull(Binary value) {
+      if (value.length() != 2) {
+        return BINARY_INVALID;
```

Comment:
Should we throw instead of returning an invalid value?

Reply:
Creates InvalidFloat16ValueException and throw it instead of an invalid value